package ddip.me.ddipbe.application.exception;

public class SuccessRecordNotFoundException extends RuntimeException {
}
